packages/nextjs/app/debug/_components/contract/DisplayVariable.tsx

```diff
@@ -44,6 +49,13 @@ export const DisplayVariable = ({
     refetch();
   }, [refetch, refreshDisplayVariables]);
 
+  useEffect(() => {
+    if (error) {
+      const parsedError = getParsedError(error);
+      notification.error(parsedError);
+    }
+  }, [error]);
```

Comment:
`onError` callback was removed in `react-query` and hence we had to do this solution, I think I found this solution somewhere in https://github.com/TanStack/query/discussions/5279